test/groovy/GaugeExecuteTestsTest.groovy

```diff
@@ -118,8 +119,8 @@ class GaugeExecuteTestsTest extends BasePiperTest {
         helper.registerAllowedMethod("sh", [String.class], { s ->
             throw new RuntimeException('Test Error')
         })
-        thrown.expect(RuntimeException)
-        thrown.expectMessage('Test Error')
+        thrown.expect(AbortException)
+        thrown.expectMessage('ERROR: The execution of the gauge tests failed, see the log for details.')
```

Comment:
Why don't you have to expect "[gaugeExecuteTests] ERROR: ..." here? (Same for the other tests) I mean, obviously the Groovy Unit Tests were successful, so I am asking out of curiosity.

Reply:
`expectMessage` checks for a substring :-)

Reply:
Sorry, could have looked that up myself.